astroquery/wfau/core.py

```diff
@@ -399,39 +441,43 @@ def get_image_list(self, coordinates, *, waveband='all', frame_type='stack',
             return request_payload
 
         response = self._wfau_send_request(query_url, request_payload)
+        self._penultimate_response = response
         response = self._check_page(response.url, "row")
+        self._last_response = response
```

Comment:
is this really necessary?

Also, try to avoid overwriting variables to help future debugging, the second `response` could be `response_row`, or whatever makes sense.

Reply:
the `_last_response` addition has historically been very useful in debugging other tools, so I prefer to keep it, but I agree about the variable renaming

Reply:
yeap, keeping `_last_response` is fine, I really just meant to spot `_penultimate_response` ;) and the rename.

Reply:
fair enough, I could drop that.  But when else do you get to use `penultimate` in the real world?!